src/components/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import "./styles/App.css";
+import PokeList from "./PokeList";
+import DetailView from "./DetailView";
+import Pokemon from "./Pokemon";
+
+function App() {
+  const [pokemon, setPokemon] = useState({
+    sprite: `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png`,
+    type: "grass",
+    id: 1,
+    name: "Bulbasaur",
+  });
```

Comment:
So this default value for useState() should more formally be treated as a constant variable. Meaning for hardcoded info like this, we would be inclined to assign it to a variable outside of the function component. Like:

```
const DEFAULT_POKEMON = {
    sprite: `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png`,
    type: "grass",
    id: 1,
    name: "Bulbasaur",
  }
  
const App = () => {
    const [...] = useState(DEFAULT_POKEMON)
    ...
}
```